docs/jwst/srctype/description.rst

```diff
@@ -92,19 +92,16 @@ background.
 Multi-Source Observations
 -------------------------
 
-NIRSpec MOS
-+++++++++++
-For NIRSpec MOS exposures (EXP_TYPE="NRS_MSASPEC"), there are multiple
-sources per exposure and hence a single user-selected parameter can't be
-used in the APT, nor a single keyword in the science product, to record the type of
-each source. For these exposures, a stellarity value can be supplied by
-the observer for each source used in the MSA Planning Tool (MPT). The
-stellarity values are
-in turn passed from the MPT to the MSA metadata (_msa.fits) file
-created by DMS and used in the calibration pipeline. The stellarity
-values from the MSA metadata file are loaded for each source/slitlet
-by the ``assign_wcs`` step of the ``calwebb_spec2`` pipeline and then
-evaluated by the ``srctype`` step to determine whether each source
+NIRSpec MOS +++++++++++ For NIRSpec MOS exposures (EXP_TYPE="NRS_MSASPEC"),
```

Comment:
About the comment on calwebb_wfs-image2 and no alias: There is no class to define an alias for. The class `Image2Pipline` already has its own alias, `calwebb_image2`. Since the alias definition is not a list, no more aliases can be defined.

If it is felt that there really has to be an alias, a placeholder class could be created that is simply a child of Image2Pipeline. 

Otherwise, the deprecation notice could be stronger, possibly removing all the information on this page except for the deprecation.